File: snippets/node/finetuning/list-finetuned-models.ts
const { CohereClient } = require('cohere-ai');

const cohere = new CohereClient({ token: process.env.COHERE_API_KEY! });

(async () => {
  const finetunedModels = await cohere.finetuning.listFinetunedModels();

  console.log(finetunedModels);
})();

File: snippets/java/app/src/main/java/finetuning/ListFinetunedModels.java
package finetuning;

import com.cohere.api.Cohere;
import com.cohere.api.resources.finetuning.finetuning.types.ListFinetunedModelsResponse;


public class ListFinetunedModels {
    public static void main(String[] args) {
        Cohere cohere = Cohere.builder().token(System.getenv("CO_API_KEY")).clientName("snippet").build();

        ListFinetunedModelsResponse response = cohere.finetuning().listFinetunedModels();

        System.out.println(response);
    }
}
